nextjs-13/src/stylesheets/components/_list-newsletter.scss

```diff
@@ -16,11 +16,23 @@
       padding: var(--size-2);
 
       color: black;
-
       cursor: pointer;
+      justify-content: space-between;
       background: var(--gray-0);
       border-radius: var(--radius-conditional-2);
       box-shadow: var(--shadow-2);
+
+      .newsletter-name {
```

Comment:
There is no need to nest this inside `newsletter-item`. Prefer to keep nesting to a minimum (i.e., you do not need to replicate the HTML structure as it can change):

```css
.list-newsletter {
  .newsletter-wrapper {}
  .newsletter-item {}
  .newsletter-name {}
  ..newsletter-button {}
}

```

> **Note**
> The team prefers using BEM (see https://nimblehq.co/compass/development/code-conventions/css/#class-names). It is not a blocker in this project but this is a heads-up as you will be asked to follow that naming on client project 😉

Reply:
👀 Ohh #til. 

Resolved [here](https://github.com/nimblehq/js-framework-benchmark/pull/116/commits/92acca69f04d80cc297861f8b3bb60060554ca9f)